src/common/Utils.ts

```diff
@@ -123,3 +123,12 @@ export function isCustomTelemetryEnabled():boolean {
         .get(CUSTOM_TELEMETRY_FOR_POWER_PAGES_SETTING_NAME);
     return isCustomTelemetryEnabled as boolean;
 }
+
+export function getUserAgent(): string {
+    const userAgent = "{product} / {product-version} {comment}";
```

Comment:
there should not be any space between the product, '/' char and version. The space after the version is significant.
"{product}/{product-version} {comment}"